### PR TITLE
Add colorblind view.

### PR DESCRIPTION
**Pull request by ancamaria99**
* Added form to choose vision type
* Here are is a screenshot of the page layout and colors for people with blue deficiency.
Screenshot 2020-09-21 at 5 52 24 PM

### CODE REVIEW

**Comment by stofee** [comment]
Tests?

Instead of a bool we should be able to select which color not to use for displaying the colors (all 120 degree part is valid use case for di_erent types of color blindness)

File: step-release-vis/src/app/components/environments/environments.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
      [ngStyle]="{position: 'relative'}"
      (mouseup)="envsMouseUp()">
   <div class="env-header"
-       [ngStyle]="{height: getTimerangeHeight() + 'px'}">
+       [ngStyle]="{height: 2 * getTimerangeHeight() + 'px'}">
```
Comment (stofee) [inline step-release-vis/src/app/components/environments/environments.html:10]: why do we need more space here?

File: step-release-vis/src/app/services/coloringService_test.ts [hunk 2]
```diff
@@ -228,4 +228,22 @@ describe('ColoringService', () => {
       expect(service.edgeOccurrences.size).toEqual(0);
     });
   });
+
+  describe('#getColorFromIndex', () => {
```
Comment (stofee) [inline step-release-vis/src/app/services/coloringService_test.ts:232]: I would expect many more cornercases to be present here, let's have at least 1 test with at least 5 candidates ;)

**Review by stofee — CHANGES_REQUESTED** [review]

File: step-release-vis/src/app/components/environments/environments.html (hunk 1 above)
Comment (laura-georgescu) [reply step-release-vis/src/app/components/environments/environments.html:10]: so we have the dark mode button underneath the vision selector, everything is in the env-header

Comment (stofee) [reply step-release-vis/src/app/components/environments/environments.html:10]: IMO side by side, just grouped together is better, we do not want to lose too many space for this ;)

Comment (laura-georgescu) [reply step-release-vis/src/app/components/environments/environments.html]: done

**Review by stofee — APPROVED** [review]

**Pull request merged**